samples/csharp_dotnetcore/49.qnamaker-all-features/BotServices.cs

```diff
@@ -34,5 +34,17 @@ private static string GetHostname(string hostname)
 
             return hostname;
         }
+
+        private static string GetAuthKey(IConfiguration configuration)
+        {
+            var authKey = configuration["QnAEndpointKey"];
+            if(string.IsNullOrEmpty(authKey))
```

Comment:
use IsNullorWhitespace

Reply:
Implemented the change.

---
In reply to: [445464918](https://github.com/microsoft/BotBuilder-Samples/pull/2536#discussion_r445464918) [](ancestors = 445464918)